external/netfx/netfx.depproj

```diff
@@ -25,5 +25,11 @@
     <FileToExclude Include="System.EnterpriseServices.Thunk" />
     <FileToExclude Include="System.EnterpriseServices.Wrapper" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">
```

Comment:
So then you want me to add these lines to the itemgroup above and remove the condition?